scripts/create-postman-collection.ts

```diff
@@ -0,0 +1,177 @@
+                    '{{API_URL}}',
+                );
+                /* eslint-disable no-param-reassign */
+                item.response[0].originalRequest.url.host = ['{{API_URL}}'];
+            }
+        }
+
+        // add the example json as the body if we have it
+        if (item.request.method === 'POST' || item.request.method === 'PUT') {
+            // TODO: better probing logic here because the -example convention only gets most files
+            // to get through I renamed some files manually
+            const exampleFilePath = path.join(examplesDir, `${resourceName}-example.json`);
+            if (fs.existsSync(exampleFilePath)) {
+                /* eslint-disable no-param-reassign */
+                item.request.body.raw = fs.readFileSync(exampleFilePath, 'utf-8');
+            } else {
+                console.log(`no file found for ${resourceName} - ${exampleFilePath}`);
+            }
+        }
+    }
+
+    if (_.has(item, 'item')) {
+        item.item.forEach((child: any) => {
+            updateRequests(child, auth, resourceName, examplesDir);
+        });
+    }
+};
+
+(async () => {
+    try {
+        console.log("let's create this thang.");
+
+        let publicCollectionPath = '';
+        let outputCollectionPath = '';
+        let examplesDir = '';
+        if (process.argv.length < 5) {
+            console.log('please pass in the file paths to your public FHIR collection, examples dir and output file');
+            process.exit(1);
+        }
+        [, , publicCollectionPath, examplesDir, outputCollectionPath] = process.argv;
+
+        // parse the public collection
+        console.log(`parsing public FHIR postman collection, ${publicCollectionPath}`);
+        const publicFHIRContents = await fs.promises.readFile(publicCollectionPath, 'utf-8');
+        const publicCollection = JSON.parse(publicFHIRContents);
+        console.log('parsing complete');
+
+        // load our collection
+        console.log('parsing fwoa postman collection');
+        const fwoaCollectionContents = await fs.promises.readFile(
+            `${__dirname}/../postman/Fhir.postman_collection.json`,
+            'utf-8',
+        );
+        const fwoaCollection = JSON.parse(fwoaCollectionContents);
+        console.log('parsing complete');
+
+        if (!_.has(fwoaCollection, 'item') || fwoaCollection.item.length === 0) {
+            console.log('fwoa postman collection does not have any items defined');
+            process.exit(1);
+        }
+
+        // pluck out the auth from an existing route
+        const fwoaAuthItem = _.find(fwoaCollection.item, item => {
+            return _.has(item, 'request') && _.has(item.request, 'auth') && item.request.auth.type !== 'noauth';
+        });
+        console.log('found fwoa auth');
+        if (_.isUndefined(fwoaAuthItem)) {
+            console.log('fwoa postman collection does not have any requests defined');
+            process.exit(1);
+        }
```

Comment:
I believe we want to log success after this check
```suggestion
        if (_.isUndefined(fwoaAuthItem)) {
            console.log('fwoa postman collection does not have any requests defined');
            process.exit(1);
        }
        console.log('found fwoa auth');
```

Reply:
Updated to use public patient and binary request definitions. So, the only remaining definitions from the existing fwoa are bundle, export and metadata.

Updated the success print statement location.